ui/components/app/confirm-gas-display/confirm-legacy-gas-display/confirm-legacy-gas-display.js

```diff
@@ -7,10 +7,10 @@ import {
   getIsMainnet,
   getIsMultiLayerFeeNetwork,
   getPreferences,
-  getUnapprovedTransactions,
   getUseCurrencyRateCheck,
   transactionFeeSelector,
   txDataSelector,
+  getUnapprovedTransactions,
```

Comment:
Non-impeditive, but in case you need to rebase before merging this change can be reverted.